security-questionnaire.md

```diff
@@ -0,0 +1,99 @@
+# [Security and Privacy Self-Review Questionnaire] for [Identifiers for WebRTC's Statistics API]
+
+## [3.1 Does this specification deal with personally-identifiable information?]
+
+Not beyond what is already exposed in [WebRTC 1.0].
+
+## [3.2 Does this specification deal with high-value data?]
+
+No
+
+## [3.3 Does this specification introduce new state for an origin that persists across browsing sessions?]
+
+No.
+
+## [3.4 Does this specification expose persistent, cross-origin state to the web?]
+
+Not beyond what is already exposed by the [WebRTC 1.0] specification itself
+
+## [3.5 Does this specification expose any other data to an origin that it doesn’t currently have access to?]
+
+The properties exposed by `RTCReceivedRTPStreamStats`, `RTCRemoteInboundRTPStreamStats`, `RTCSentRTPStreamStats`, `RTCOutboundRTPStreamStats`, `RTCRemoteOutboundRTPStreamStats`, `RTCIceCandidatePairStats`, `RTCTransportStats` expose network-layer data not currently available to the JavaScript layer.
+
+In addition, the behavior of WebRTC Stats (in particular `RTCMediaStreamTrackStats`) on [isolated media streams] (which aren't otherwise available to the JavaScript layer) is currently ill-defined.
+
+## [3.6 Does this specification enable new script execution/loading mechanisms?]
+
+No.
+
+## [3.7 Does this specification allow an origin access to a user’s location?]
+
+Beyond the risks associated with IP addresses as discussed in the WebRTC 1.0 specification, some combination of some of the network properties uniquely exposed by this specification can be correlated with  location.
+
+For instance, the [round-trip time] exposed in `RTCRemoteInboundRTPStreamStats` can give some coarse indication on how far aparts the peers are located, and thus, if one of the peer's location is known, this may reveal information about the other peer.
```

Comment:
There are many ways the latency can be calculated, so I do not know specifically calling out RTT makes sense here.
see https://w3c.github.io/resource-timing/#introduction, shows how you can use resource timing API to calculate latency using HTTP.

Reply:
The RTT gives you a metric for client-to-client distance, while HTTP just gives you client-to-server distance. That might matter a little.
You can always measure RTT using datachannel (just send a message and have it returned), so hiding RTT doesn't mean RTT can't be observed (or at least given an upper bound).

Reply:
@alvestrand that is correct, RTT can be observed by implementing a ping mechanism within the datachannel.